siri-salay/week-1/monument-mockup/pt-1/starter-code/css/main.css

```diff
@@ -0,0 +1,229 @@
+
+body {
+  width:70em;
+  margin:0 auto;
+  color: #43545d;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: row;
+  margin-top: 3em;
+}
+
+.header{
+  width: 70em;
+  margin: 0 auto;
+}
+
+.main {
+  width: 90em;
+  margin: 0 3em;
+}
+
+
+.sidebar {
+  margin-top: 30px;
+  margin-left: 3em;
+  width: 35em;
+}
+
+.about h2 {
+  font-size: 14px;
+}
+
+.mainImg {
+  width: 100%;
+}
+
+.aboutImg {
+  width: 90%;
+}
+
+.hamburger {
+  display: none;
+}
+
+nav ul{
+  width: 50em;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin: 0 auto;
+  height: 7em;
+  width: 70em;
+  padding-top: 3.5em;
+  line-height: 1em;
+  top: 0;
+  position: relative;
+  background-color: white;
+  z-index: 100;
+}
+
+nav ul li {
+    list-style-type: none;
+}
+
+nav ul li a {
+  color: #8ba6b4;
+}
+
+ul li {
+color: #8ba6b4;
+}
+
+h1 {
+  position: relative;
+  top: 6em;
+  letter-spacing: 11px;
+  font-size: 32px;
+  color: white ;
+  font-weight: normal;
+}
+
+
+h1 {
+  text-align: center;
+  padding-bottom: 1em;
+  line-height: 22px;
+}
+
+h2, h3 {
+  letter-spacing: 3px;
+}
+
+.main h2 {
+  margin-bottom: 2em;
+}
+
+
+h3 {
+  font-size: 14px;
+}
+
+h2 {
+  font-size: 18px;
+}
+
+p {
+  margin: 20px auto;
+  color: #808c91;
+  font-size: 12px;
+  line-height: 17px;
+  font-family: "Raleway", sans-serif;
+}
+
+footer {
+  background: #c9f3f6;
+  text-align: center;
+  height: 20em;
+  margin: 4em 0;
+  display: flex;
+  flex-direction: column;
+  padding: 4em 0;
+}
+
+footer nav {
+display: flex;
+flex-direction: row;
+justify-content: center;
+align-items: center;
+}
+
+footer nav li
+{
+  margin-top: 2em;
+}
+
+
+footer nav li i{
+ width: 3em;
+ height: 3em;
+ font-size: 32px;
+}
+
+input[type="email"]{
+  background-color: white;
+  border-radius: 0;
+  margin-right: 20px;
+  text-align: center;
+  padding: 10px 5px;
+  border: none;
+  width: 16em;
+  height: 3em;
+
+  margin-top: 2em;
+}
+
+input[type="submit"] {
+  background-color: transparent;
+margin-left: 40px;
+width: 20em;
+height: 3em;
+
+
+  margin-top: 2em;
+
```

Comment:
A few recommendations to keep in mind:
* Keep clean whitespace (extra lines, consistent tabs/spaces, brackets closing at the same indentation they were opened, etc.)
* Commenting! It helps both the people reviewing your code figure out at a glance what's going on and also future-Siri will have an easier time getting a grasp on each section. Comments that answer questions such as:
   * What is the responsibility of this chunk of code?
   * Will this logic or styling be changed later? How?
   * What is yet to be built?